packages/sql/src/RequestManager.ts

```diff
@@ -159,7 +166,8 @@ export class RequestManager {
         if (!data.error) {
           resolve(data);
         } else {
-          reject(data.error);
+          const message = data && data.error && data.error.length ? data.error[0] : 'Unknown error';
```

Comment:
Hopefully some chaining soon, if we upgrade Typescript #23

Reply:
I forgot, yes. But I think it works. Let's try

Reply:
Not a priority though, it just came to my mind 😄